cli/cmd/stat.go

```diff
@@ -413,6 +453,12 @@ func printStatJSON(statTables map[string]map[string]*row, w *tabwriter.Writer) {
 					entry.LatencyMSp95 = &stats[key].latencyP95
 					entry.LatencyMSp99 = &stats[key].latencyP99
 					entry.TLS = &stats[key].tlsPercent
+
+					if options.showTCPStats {
```

Comment:
I'm actually wondering whether we should just query tcp stats by default if `-o json` is specified, so that we don't have these weird null values

Reply:
I was thinking maybe `-o wide` and `-o json`? I'd rather manage it that way than a new flag. In the future, we'll be able to do something like `kubectl` does with custom columns.

Reply:
yeah I think auto including it with `-o wide` makes sense

Reply:
I think I'm agreeing with what you're both saying but I think we should always include all fields in json output, since width is not really a concern.